test/fastifyCsrf.test-d.ts

```diff
@@ -0,0 +1,26 @@
+import * as fastify from "fastify";
```

Comment:
Please update the code style as specified in comment above.

Reply:
Done, see 2b82425